### PR TITLE
 👍 Corrected the spelling mistake in TopView from Multiple FLightpath…

### PR DESCRIPTION
**Pull request by ambiguousphoton**
I have corrected the spelling mistake in TopView from "Multiple FLightpath" to "Multiple Flightpath".
Please merge this.

### CODE REVIEW

**Comment by ambiguousphoton** [comment]
ok

File: NOTICE [hunk 1]
```diff
@@ -113,13 +113,13 @@ Further Information: http://tango.freedesktop.org
 Airports Data
 -------------
 
-To draw airports on the topview we use the services provided by ourairports
+To draw airports on the w we use the services provided by ourairports
```
Comment (joernu76) [inline NOTICE:116]: This doesn't look right... Same below.

**Review by joernu76 — CHANGES_REQUESTED** [review]
Please fix the newly introduced spelling mistake in NOTICE.

**Comment by ambiguousphoton** [comment]
Ok

**Pull request closed without merging**